docs/topics/db/transactions.txt

```diff
@@ -252,6 +252,130 @@ by Django or by third-party libraries. Thus, this is best used in situations
 where you want to run your own transaction-controlling middleware or do
 something really strange.
 
+Performing actions after commit
+===============================
+
+.. versionadded:: 1.9
+
+Sometimes you need to fire off an action related to the current database
+transaction, but only if the transaction successfully commits. Examples: a
```

Comment:
The sentence fragment here was lazy writing on my part. Better would be "Examples might include a..."

I also might lean towards taking out the specific reference to Celery here, but I don't feel strongly either way.